src/Components/Common/SearchBox/SearchBox.scss

```diff
@@ -3,16 +3,16 @@
 .searchbox {
   background: $color-light-soft;
   border: 0;
-  border-radius: 3px;
-
-  .dropdown-toggle {
-    width: 100px !important;
-  }
-
+  border-radius: 4px;
+  display: flex;
+  padding-top: 0 !important;
   input {
     border: 0px solid rgba(0, 0, 0, 0.1);
     background: $color-light-soft;
-    width: 300px !important;
+  }
+
+  .input-group {
+    padding: 0 !important;
   }
   .btn-outline-primary {
     color: $color-primary;
```

Comment:
@coolbeatz71 Could you remove the `outline` on these 3 properties: The search box, the drop-down, the search button, and make the search box a little bit wider as we previously designed.

Reply:
Yes @gracelungu I have removed it